hatch_pip_compile/plugin.py

```diff
@@ -260,18 +269,10 @@ def piptools_constraints_file(self) -> Optional[pathlib.Path]:
         """
         Get default lock file
```

Comment:
Could take this opportunity to not say "default" in this comment because we have since decided on a different approach